### ISSUE TITLE
Facing low PCC in effective_se_module of VoVNet

### ISSUE THREAD

**Issue by keerthana-r-mcw** [tenstorrent/tt-metal#13313]
**Describe the bug**
Pcc is dropping for effective_se_module of VoVNet model.

**To Reproduce**
1. Checkout [keerthanar/vovnet_ttnn](https://github.com/tenstorrent/tt-metal/tree/keerthanar/vovnet_ttnn)
2. Run `pytest /tt-metal/models/experimental/functional_vovnet/tests/test_ttnn_functional_vovnet.py::test_effective_se_module`
3. The following will occur,
`E       AssertionError: -0.0016499053079912294`

**Please complete the following environment information:**
- Device: WH n150

**Comment by Sudharsan-V** [comment]
Yes @keerthana-r-mcw, with the random weights, the pcc is good. When the pretrained weights are used, pcc drops.

The issue is explained [here (#18137)](https://github.com/tenstorrent/tt-metal/issues/18137#issuecomment-2678262003).
Closing this ticket.
